src/test/java/uk/gov/hmcts/reform/et/syaapi/controllers/ManageCaseControllerTest.java

```diff
@@ -171,4 +189,53 @@ void shouldCreateDraftCase() throws Exception {
             .andExpect(jsonPath("$.created_date").exists())
             .andExpect(jsonPath("$.last_modified").exists());
     }
+
+    @Test
+    void shouldStartUpdateCase() throws Exception {
+        /*CaseDataContent caseDataContent = CaseDataContent.builder()
+            .event(Event.builder().id(String.valueOf(CaseEvent.UPDATE_CASE_DRAFT)).build())
+            .eventToken(startEventResponse.getToken())
+            .data(caseData)
+            .build();*/
```

Comment:
Good practice should remove any commented out code

Reply:
code review comments incorporated.